discussion_agents/cog/prompts/react.py

```diff
@@ -74,3 +74,498 @@
+Thought 2: Because he "appeared in the 2009 Fox television film Virtuality", he should have worked with the Fox Broadcasting Company.
+Action 2: Finish[SUPPORTS]
+
+Claim: Stranger Things is set in Bloomington, Indiana.
+Thought 1: I should search for Stranger Things, and see if it is set in Bloomington, Indiana.
+Action 1: Search[Stranger Things]
+Observation 1: Stranger Things is an American science fiction horror drama television series created by the Duffer Brothers. Set in the 1980s, primarily in the fictional town of Hawkins, Indiana, the series centers on a number of mysteries and supernatural events occurring around the town and their impact on an ensemble of child and adult characters. 
+Thought 2: The observation says that it is set in a "fictional town of Hawkins, Indiana", so it is not set in Bloomington.
+Action 2: Finish[REFUTES]
+
+Claim: Beautiful reached number two on the Billboard Hot 100 in 2003.?
+Thought 1: I need to search the song Beautiful and find if it reached number two on the Billboard Hot 100 in 2003.
+Action 1: Search[Beautiful]
+Observation 1: Could not find [Beautiful]. Similar: ['Beautiful', 'Beautiful, Beautiful', 'A Beautiful Mind (film)', 'Beautiful (Christina Aguilera song)', 'Life Is Beautiful'].
+Thought 2: From suggestions, I should search "Beautiful (Christina Aguilera song)" to find the song.
+Action 2: Search[Beautiful (Christina Aguilera song)]
+Observation 2: "Beautiful" is a song recorded by American singer Christina Aguilera for her fourth studio album, Stripped (2002).
+Thought 3: It does not mention Billboard, so I need to look up "Billboard Hot 100" to find if it reached number two on it in 2003.
+Action 3: Lookup[Billboard Hot 100]
+Observation 3: (Result 1 / 3) The song peaked at number two on the Billboard Hot 100 in the United States, where it was certified Gold for 500,000 units shipped.
+Thought 4: It only says the song peaked at number two on the Billboard Hot 100, but not if it was in 2003. I am not sure if this claim is true or not.
+Action 4: Finish[NOT ENOUGH INFO]
+"""
+
+
+REACT_ALFWORLD_INSTRUCTION = """
+{examples}
+{question}
+{scratchpad}
+"""
+
+REACT_ALFWORLD_PROMPTS_EXAMPLE  = {"react_put_0": 
```

Comment:
Damn, this is a giant dictionary. Is there any way to format this dictionary better for readability?